lib/score.rb

```diff
@@ -39,10 +39,10 @@ def self.score(word)
         word_score += value[letter]
```

Comment:
The `value` hash above will be re-created every time the `score` method is called which seems redundant. I think it would be better if this hash was stored as a constant because then it would not be recreated each time the method was called.